### PR TITLE
refactor: consolidate open links fns

### PR DESCRIPTION
**Pull request by setchy**

### CODE REVIEW

File: src/utils/links.test.ts [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+// export function openDeveloperSettings(account: Account) {
+//   const url = getDeveloperSettingsURL(account);
+//   openExternalLink(url);
+// }
+
+// export async function openNotification(notification: Notification) {
+//   const url = await generateGitHubWebUrl(notification);
+//   openExternalLink(url);
+// }
```
Comment (afonsojramos) [inline src/utils/links.test.ts:118]:
```suggestion
```

**Review by bmulholland — APPROVED** [review]
Love it!

File: src/utils/links.test.ts [hunk 2]
```diff
@@ -0,0 +1,118 @@
+import { partialMockUser } from '../__mocks__/partial-mocks';
```
Comment (afonsojramos) [inline src/utils/links.test.ts:1]: Almost unnecessary to test this file, but fine 😅

**Review by afonsojramos — APPROVED** [review]

File: src/utils/links.test.ts (hunk 2 above)
Comment (setchy) [reply src/utils/links.test.ts:1]: i know right....  trying to make the tests a little less entangled... one test case at a time

**Pull request merged**